content/workshops/cnd-deploy-practices.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Building and deploying a Cloud Native Application"
+draft: false
+description: >
+    You will build and deploy a web application on a modern container
+    orchestration platform,
+    and learn cloud native principles along the way.
+summary:
+    You will build and deploy a web application on a modern container
+    orchestration platform,
+    and learn cloud native principles along the way.
+topics:
+  - Practices
```

Comment:
> topics:
>  - Practices

This is what's breaking the link check, as "Practices" isn't an existing [topic](https://tanzu.vmware.com/developer/topics/). I thought we had a test to check to make sure that any topic defined in the frontmatter matches an existing topic, but turns out that's in the backlog currently.

Reply:
@BrianMMcClain thanks for the clarification.  

If I understand below this is not the root cause of the broken link check in this build, but may be in a future build after the test is implemented.  Can you confirm whether this will be problem for this build, or going forward?

The **Practices** topic is relevant for the series of workshops be are building now, so IMO it should be added to the Development Topics list.

Reply:
So it is actually the cause of the broken link. When the site builds the `/topics` page, it looks at the `topics` section of all of the existing content (guides, blogs, workshops, samples, etc) to figure out what topics exist. While building that page, it looks for an icon to accompany it of the same name to fill in the design of how topics are listed. Finally, it also creates a topic page of the same name, which also looks for the same icon.

In this case, since the "Practices" topic doesn't currently exist, the icon doesn't exist either, which we can see it's looking for in the logs of the link check test which is showing there's an SVG missing on the `/topics` and the `/topics/practices` pages, which doesn't exist, resulting in a 404.


```
http://localhost:1313/developer/topics/
	404	http://localhost:1313/developer/images/icons/icon-%25!s%28%3cnil%3e%29.svg
http://localhost:1313/developer/topics/practices/
	404	http://localhost:1313/developer/images/icons/icon-%25!s%28%3cnil%3e%29.svg
```

So the two options are:

1. Set the value of the `topics` frontmatter to one of the existing topics
2. Create a `practices` topic

We _can_ create a practices topic, but our topics are generally pretty static, and any grouping of content are either done by tags. The other thing I'll bring up is that we're also working on introducing the concept of a "series", which walks a user through a collection of guides, workshops, videos, etc with an overarching narrative to accompany them. On the discussion of creating a new topic, I'll reach out so we can better discuss and figure out what works best for everyone :)

Reply:
@BrianMMcClain Thanks, I understand the full picture.

Is it possible to publish the workshop without the topic?  

The existing topics do not fit, 
and using one arbitrarily might mislead  the audience as to the intent of the workshop.

I do think having a discussion on additional topics would be a great idea.
_Practices_ might be too general, and I have some thoughts in thinking 
ahead the workshops we currently plan to build.

But we do want to get the workshop published for partner review as soon as practical,
even if not perfect.

How about we touch beginning of next week?

Reply:
@BrianMMcClain Regarding the _series_ topic, I look forward to that, because we are thinking there are clear linear relationships between the workshops we are currently building, as well as some of the other product-centric and practices content.

It is not a show-stopper for us at this point - we are calling out links explicitly in the workshop for other content where it makes sense.  The first workshop is ready to go, and we have additional 2 workshops in flight we plan for first draft publishing in a couple of weeks.

Perhaps I can walk you through to demonstrate some examples, and after the series features are in place I think (hope) we can do some gentle refactoring to accommodate the new series features.